line/__tests__/end_to_end_unit.test.mjs

```diff
@@ -1,39 +1,39 @@
-import lineRouter from '../index.mjs';
-import express from 'express';
-import request from 'supertest';
+import lineRouter from '../index.mjs'
```

Comment:
The semicolons are gone and back? You need to make sure you have your workspace settings consistent.

Reply:
Thanks updated in the code